src/test/kotlin/lotto/LottoTest.kt

```diff
@@ -0,0 +1,68 @@
+package lotto
+
+import io.kotest.matchers.shouldBe
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.CsvSource
+import org.junit.jupiter.params.provider.ValueSource
+
+class LottoTest {
+
+    @Test
+    fun `1 ~ 45 까지의 로또 번호에서 랜덤한 숫자 하나를 뽑을 수 있다`() {
+        val actual = LottoNumber().getLottoNumber()
+        assertThat(actual).isBetween(1, 45)
+    }
+
+    @Test
+    fun `1000원으로 한개의 로또를 발급 받을 수 있다`() {
+        val actual = Lotto().buyLotto(1000)
+        assertThat(actual.size).isEqualTo(1)
+    }
+
+    @Test
+    fun `한 개의 로또는 총 6개의 숫자로 이루어져 있다`() {
+        val actual = Lotto().buyLotto(1000)
+        assertThat(actual[0].size).isEqualTo(6)
+    }
+
+    @Test
+    fun `발급 받은 6개의 숫자 중 중복 된 숫자는 있을 수 없다`() {
+        val lotto = Lotto().buyLotto(1000)
+        val actual = lotto[0].toSet()
```

Comment:
```suggestion
        val actual = lotto[0]
```
set으로 변환하는 순간 중복된 숫자가 없어지게 됩니다.
정말로 중복된 숫자가 있는지 확인해보기 위해서는 set으로 변환하지 않도록 개선해보면 어떨까요?

Reply:
해당 부분에서 질문이 있습니다!

먼저 제가 의도한 바는 이렇습니다.

검증하고자 하는 것 - 발급 받은 6개의 숫자 중 중복 된 숫자는 있을 수 없다 

즉, buyLotto()를 통해 얻은 lotto는 무조건 6개의 숫자로 이루어진 리스트여야 한다 -> toSet()을 하게 되면 중복된 숫자가 없어지게 된다 -> 그래서 toSet()을 하고 이후에 검증하고자하는 assert문에서 6이 나오지 않으면 검증하고자 했던 buyLotto()는 실패한다. -> toSet()을 했는데도 6이 나오면 중복된 것이 없다는 의미이기 때문에 테스트가 통과한다.

라는 의미로 작성하였습니다.

혹여 틀린 부분있다면 답변 부탁드리겠습니다! :)

Reply:
말씀하신 의도라면 set으로 변환하는 것이 의미가 있겠네요! 👍